systemd/openqa-dehydrated.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Renewal openQA certificates.
+After=network.target nginx.service
+Requires=network.target
+Wants=openqa-setup-db.service
```

Comment:
Why does it depend on openqa-setup-db?

Reply:
copied by another file initially. no need. I will remove it